test/views/businessactivities/expected_amls_turnoverSpec.scala

```diff
@@ -60,7 +59,7 @@ class expected_amls_turnoverSpec extends AmlsViewSpec with MustMatchers  {
 
       val form2: ValidForm[ExpectedAMLSTurnover] = Form2(ExpectedAMLSTurnover.Third)
 
-      def view = views.html.businessactivities.expected_amls_turnover(form2, true, businessMatching2)
+      def view = views.html.businessactivities.expected_amls_turnover(form2, true, None, Some(List("accountancy service provider", "estate agency business")))
```

Comment:
Any reason you are replacing businessMatching2 with None?

Reply:
I'm not testing businessMatching2 anymore I'm testing on the optional list I have provided